src/material-experimental/mdc-tabs/tab-group.spec.ts

```diff
@@ -756,6 +757,100 @@ describe('MDC-based MatTabGroup', () => {
     }));
   });
 
+  describe('tabs with custom css classes', () => {
+    let fixture: ComponentFixture<TabsWithClassesTestApp>;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(TabsWithClassesTestApp);
+    });
+
+    it('should apply label classes', () => {
+      fixture.detectChanges();
+
+      const labelElements = fixture.debugElement
+        .queryAll(By.css('.mdc-tab.hardcoded.label.classes'));
```

Comment:
Usually the way we assert cases like this is to query for the tab and then check `expect(tab.classList).toContain('foo')`. This makes the failure message easier to read.